jwst/pipeline/calwebb_spec2.py

```diff
@@ -86,10 +87,7 @@ def process(self, input):
             )
 
             # Save result
-            suffix = 'cal'
-            if isinstance(result, datamodels.CubeModel):
-                suffix = 'calints'
-            self.save_model(result, suffix)
+            self.save_model(result)
```

Comment:
This looks like you're changing the interface to `save_model`, but I don't see an update to `save_model` in here. Will `save_model` work without supplying the `suffix` argument? Maybe you meant to change the call to just use the regular `save` function?

Reply:
Nope. This is intended interface usage.  When no suffix is supplied, the `Step.suffix` is used.  The rest of the removed code is actually duplicate of what is done in the `process_exposure_product` method.